diff --git a/conf/battle/player.conf b/conf/battle/player.conf
index 5dcf10930..207d014a7 100644
--- a/conf/battle/player.conf
+++ b/conf/battle/player.conf
@@ -66,16 +66,16 @@ natural_heal_weight_rate: 50
 natural_heal_weight_rate_renewal: 70
 
 // Maximum atk speed. (Default 190, Highest allowed 199)
-max_aspd: 190
+max_aspd: 199
 
 // Same as max_aspd, but for 3rd classes. (Default 193, Highest allowed 199)
-max_third_aspd: 193
+max_third_aspd: 199
 
 // Max ASPD for extended class (Kagerou/Oboro and Rebellion). (Default 193, Highest allowed 199)
-max_extended_aspd: 193
+max_extended_aspd: 199
 
 // Max ASPD for Summoner Class (Doram). (Default 193, Highest allowed 199)
-max_summoner_aspd: 193
+max_summoner_aspd: 199
 
 // Maximum walk speed rate (200 would be capped to twice the normal speed)
 max_walk_speed: 300
