diff --git a/conf/char_athena.conf b/conf/char_athena.conf
index 726dd3077..f7032ac45 100644
--- a/conf/char_athena.conf
+++ b/conf/char_athena.conf
@@ -214,7 +214,7 @@ db_path: db
 // NOTE: Requires client 2011-03-09aragexeRE or newer.
 // A window is opened before you can select your character and you will have to enter a pincode by using only your mouse.
 // Default: yes
-pincode_enabled: yes
+pincode_enabled: no
 
 // How often does a user have to change his pincode?
 // 0: never (default)
