diff --git a/conf/battle/drops.conf b/conf/battle/drops.conf
index d4b876f11..6c897a1a3 100644
--- a/conf/battle/drops.conf
+++ b/conf/battle/drops.conf
@@ -38,50 +38,50 @@ mvp_item_third_get_time: 2000
 // Item drop rates (Note 2)
 
 // The rate the common items are dropped (Items that are in the ETC tab, besides card)
-item_rate_common: 100
-item_rate_common_boss: 100
-item_rate_common_mvp: 100
-item_drop_common_min: 1
+item_rate_common: 1000
+item_rate_common_boss: 1000
+item_rate_common_mvp: 1000
+item_drop_common_min: 100
 item_drop_common_max: 10000
 
 // The rate healing items are dropped (items that restore HP or SP)
-item_rate_heal: 100
-item_rate_heal_boss: 100
-item_rate_heal_mvp: 100
-item_drop_heal_min: 1
+item_rate_heal: 1000
+item_rate_heal_boss: 1000
+item_rate_heal_mvp: 1000
+item_drop_heal_min: 100
 item_drop_heal_max: 10000
 
 // The rate at which usable items (in the item tab) other then healing items are dropped.
-item_rate_use: 100
-item_rate_use_boss: 100
-item_rate_use_mvp: 100
-item_drop_use_min: 1
+item_rate_use: 1000
+item_rate_use_boss: 1000
+item_rate_use_mvp: 1000
+item_drop_use_min: 100
 item_drop_use_max: 10000
 
 // The rate at which equipment is dropped.
-item_rate_equip: 100
-item_rate_equip_boss: 100
-item_rate_equip_mvp: 100
-item_drop_equip_min: 1
+item_rate_equip: 1000
+item_rate_equip_boss: 1000
+item_rate_equip_mvp: 1000
+item_drop_equip_min: 200
 item_drop_equip_max: 10000
 
 // The rate at which cards are dropped
-item_rate_card: 100
-item_rate_card_boss: 100
-item_rate_card_mvp: 100
-item_drop_card_min: 1
+item_rate_card: 1000
+item_rate_card_boss: 1000
+item_rate_card_mvp: 1000
+item_drop_card_min: 200
 item_drop_card_max: 10000
 
 // The rate adjustment for the MVP items that the MVP gets directly in their inventory
 // Mode: 0 - official order, 1 - random order, 2 - all items
-item_rate_mvp: 100
-item_drop_mvp_min: 1
+item_rate_mvp: 1000
+item_drop_mvp_min: 100
 item_drop_mvp_max: 10000
 item_drop_mvp_mode: 0
 
 // The rate adjustment for equip-granted item drops.
 item_rate_adddrop: 100
-item_drop_add_min: 1
+item_drop_add_min: 100
 item_drop_add_max: 10000
 
 // The rate adjustment for items inside of equip-granted item group drops.
@@ -91,8 +91,8 @@ item_group_drop_min: 1
 item_group_drop_max: 10000
 
 // Rate adjustment for Treasure Box drops (these override all other modifiers)
-item_rate_treasure: 100
-item_drop_treasure_min: 1
+item_rate_treasure: 1000
+item_drop_treasure_min: 100
 item_drop_treasure_max: 10000
 
 // Use logarithmic drops? (Note 1)
