diff --git a/conf/battle/exp.conf b/conf/battle/exp.conf
index 90a33dec9..479af3908 100644
--- a/conf/battle/exp.conf
+++ b/conf/battle/exp.conf
@@ -10,10 +10,10 @@
 //--------------------------------------------------------------
 
 // Rate at which exp. is given. (Note 2)
-base_exp_rate: 100
+base_exp_rate: 5000
 
 // Rate at which job exp. is given. (Note 2)
-job_exp_rate: 100
+job_exp_rate: 5000
 
 // Turn this on to allow a player to level up more than once from a kill. (Note 1)
 multi_level_up: no
