diff --git a/conf/login_athena.conf b/conf/login_athena.conf
index 734d3bdbc..f4407db2c 100644
--- a/conf/login_athena.conf
+++ b/conf/login_athena.conf
@@ -53,7 +53,7 @@ console: off
 
 // Can you use _M/_F to make new accounts on the server?
 // Note: This only works if client side password encryption is not enabled.
-new_account: no
+new_account: yes
 
 // If new_account is enabled, changes the minimum length for the account name.
 // By default is set to '4' or '6' (depending on the new login UI).
